packages/jest/src/migrations/update-13-1-2/update-tsconfigs-for-tests.ts

```diff
@@ -0,0 +1,111 @@
+import { forEachExecutorOptions } from '@nrwl/workspace/src/utilities/executor-options-utils';
+import { JestExecutorOptions } from '@nrwl/jest/src/executors/jest/schema';
+import {
+  formatFiles,
+  joinPathFragments,
+  logger,
+  readProjectConfiguration,
+  stripIndents,
+  Tree,
+} from '@nrwl/devkit';
+
+function updateTsConfigsForTests(tree: Tree) {
+  forEachExecutorOptions<JestExecutorOptions>(
+    tree,
+    '@nrwl/jest:jest',
+    (jestOptions, projectName) => {
+      const projectConfig = readProjectConfiguration(tree, projectName);
+
+      const tsconfigSpecPath = joinPathFragments(
+        projectConfig.root,
+        'tsconfig.spec.json'
+      );
+
+      if (!tree.exists(tsconfigSpecPath)) {
+        return;
+      }
+
+      updateTsConfigInclude(tree, tsconfigSpecPath);
+
+      switch (projectConfig.projectType) {
+        case 'library':
+          const tsConfigLibPath = joinPathFragments(
+            projectConfig.root,
+            'tsconfig.lib.json'
+          );
+          updateTsConfigExclude(tree, tsConfigLibPath);
+          break;
+        case 'application':
```

Comment:
There's going to be a bunch of `applications` that do not have `tsconfig.app.json` because it is an e2e project. Could you please account for this in your logic or... don't `logger.error` when you can't find a tsconfig?